# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tvtime/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Naur tvtime-1.0.2.org/src/xmltv.c tvtime-1.0.2/src/xmltv.c
--- tvtime-1.0.2.org/src/xmltv.c	2009-07-02 21:48:49.426191088 +0200
+++ tvtime-1.0.2/src/xmltv.c	2009-07-02 21:50:20.842066085 +0200
@@ -118,9 +118,9 @@
 typedef struct {
     const char *code;
     const char *name;
-} locale_t;
+} tvtime_locale_t;
 
-static locale_t locale_table[] = {
+static tvtime_locale_t locale_table[] = {
     {"AA", "Afar"},           {"AB", "Abkhazian"},      {"AF", "Afrikaans"},
     {"AM", "Amharic"},        {"AR", "Arabic"},         {"AS", "Assamese"},
     {"AY", "Aymara"},         {"AZ", "Azerbaijani"},    {"BA", "Bashkir"},
@@ -168,7 +168,7 @@
     {"XH", "Xhosa"},          {"YO", "Yoruba"},         {"ZH", "Chinese"},
     {"ZU", "Zulu"} };
 
-const int num_locales = sizeof( locale_table ) / sizeof( locale_t );
+const int num_locales = sizeof( locale_table ) / sizeof( tvtime_locale_t );
 
 /**
  * Timezone parsing code based loosely on the algorithm in

